aperturedb/BlobDataCSV.py

```diff
@@ -66,7 +66,6 @@ def get_indices(self):
         }
 
     def getitem(self, idx):
-        idx = self.df.index.start + idx
         filename = os.path.join(self.relative_path_prefix,
                                 self.df.loc[idx, BLOB_PATH])
```

Comment:
The removed line is setting `idx`, but the new line 70 is still referencing `idx`

Reply:
idx is a param of the method.

Reply:
right, but I would assume there was a reason idx was set there to begin with?